static/src/stylesheets/base/_helpers.scss

```diff
@@ -268,3 +268,21 @@
         vertical-align: middle;
     }
 }
+
+/**
+ * Nested links
+ *
+ * Support nested links by wrapping inner links in <object> element
+ * We need to hide the object element to prevent layout thrashing
+ * http://kizu.ru/en/fun/nested-links/
+ */
+
+.u-nested-link {
+    width: 0;
```

Comment:
should these `display: none` in old-ie?

Reply:
Since CSS doesn't seem to load (or get applied) at all in <IE9, I don't think there's much point in worrying about it?